src/coreclr/jit/emit.h

```diff
@@ -2397,6 +2398,7 @@ class emitter
     void emitAdvanceInstrDesc(instrDesc** id, size_t idSize) const;
     size_t emitIssue1Instr(insGroup* ig, instrDesc* id, BYTE** dp);
     size_t emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp);
+    BYTE* emitInstrSve(instrDesc* id, BYTE* dst);
```

Comment:
rename it to `emitOutputInstrSve()` . Also should be guarded by `#ifdef TARGET_ARM64`.

Reply:
I originally wanted to name it that, but `emitOutputInstrSve` can't be called independently of `emitOutputInstr`, and the signature is different, such as `emitOutputInstr` returns a `size_t` where as the `emitOutputInstrSve` would return a `BYTE*`.

Maybe I'm being too picky on the name. I noticed we also have a `emitOutput_Instr` which has a similar signature to `emitInstrSve`. Maybe I can just call it `emitOutput_InstrSve` ... just putting an underscore in there.

Reply:
Yea I wasn't thinking, this should have just been defined in `emitarm64.h` rather than in `emit.h`.